ctype.md

```diff
@@ -56,7 +54,7 @@ Open it and paste the following:
 ```json
 {
   "schema": {
-    "$id": "KILT_WORKSHOP_ATTENDED",
+    "$id": "SimpleLicense",
```

Comment:
As mentioned, we are going to be adding changes with the nested CTypes. 

I believe we will update this afterwards.

Reply:
Agreed 👍